### PR TITLE
Clean Twig files formatting

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Clean Twig files formatting
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | @PrestaShopCorp
| How to test?      | Check if the module still works

### CODE REVIEW

**Comment by ps-jarvis** [comment]
This pull request seems to contain new translation strings. I have summarized them below to ease up review:

- `Admin.Actions`
    - [ ] `Go back`

(Note: this is an automated message, but answering it will reach a real human)

**Review by ga-devfront — APPROVED** [review]
image

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
